lib/neo-test-functions.js

```diff
@@ -51,6 +51,32 @@
     return particle
   }
 
+  // Get the index of the AC
+  // This is the case with multiple ACs
+  function getAcIndex(path) {
+    let adaptiveCardIndex = 0
+    if (Number.isInteger(parseInt(path[0]))) {
+      adaptiveCardIndex = path.shift()
+    }
+    return adaptiveCardIndex
+  }
+
+  // Construct path to a nested AC object
+  // Uses the AC Index and the path
+  function constructPath(adaptiveCardIndex, path) {
+    return `restrictedParticle.response.content[${adaptiveCardIndex}].${path.join('.')}`
+  }
+
+  // Get data nested in an AC
+  // Uses the path to the data/object
+  function getAcData(path, particle) {
+    const restrictedParticle = particle
+    const adaptiveCardActionIndex = getAcIndex(path)
+    const actionDataPath = constructPath(adaptiveCardActionIndex, path)
```

Comment:
Any need why this is put into a separate (1-line) function?

Reply:
It was in use more than one time before...fixed :+1: 
https://github.com/neohelden/test-automation/pull/6/files#diff-f55cc924b5da9f0599b41b7520468148d530341b040fbba6b27cd7f14994055aR59